roles/lnls-ans-role-conda/vars/Debian-buster.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
If all of the debian files are exactly the same we can just use a single `Debian.yml` file. Ansible will use it for all Debian-based distros